engine/src/main/java/org/destinationsol/CommonDrawer.java

```diff
@@ -18,45 +18,58 @@
 import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.graphics.Color;
 import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.OrthographicCamera;
 import com.badlogic.gdx.graphics.g2d.BitmapFont;
 import com.badlogic.gdx.graphics.g2d.GlyphLayout;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
 import com.badlogic.gdx.graphics.g2d.TextureRegion;
 import com.badlogic.gdx.math.Matrix4;
 import com.badlogic.gdx.math.Rectangle;
 import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.utils.viewport.ScreenViewport;
+import com.badlogic.gdx.utils.viewport.Viewport;
 import org.destinationsol.assets.Assets;
 import org.destinationsol.common.SolMath;
+import org.destinationsol.ui.DimensionsRatio;
 import org.destinationsol.ui.UiDrawer;
 
 public class CommonDrawer {
-    public final float width;
-    public final float height;
-    public final float dimensionsRatio;
+    public DimensionsRatio dimensionsRatio;
+    public float width;
+    public float height;
+    public float r;
```

Comment:
radius? record? resolution? root? ransomware? rhinoceros? could even be ratio but who really knows